components/Header.tsx

```diff
@@ -1,4 +1,4 @@
-import { Link } from "aleph/react";
+import { Link } from 'aleph/react';
```

Comment:
can you please run `deno fmt`? thanks

Reply:
@ije I have formatted the code, also if any other frontend or docs tasks you want to let me know, I really love working with alephjs and deno, so we can create a great tooling system for deno web environment.

Reply:
thanks! unfortunately currently i don't have too much time on this project...